test/atm/test_convert_gsi_satbias.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -ex
```

Comment:
still need -x?

Reply:
I thought it might be useful for debugging purposes